frontend/src/components/TeacherTitle.jsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import { Link } from 'react-router-dom';
+
+import ArrowBack from '../assets/arrow-back.svg';
+
+const TeacherTitleWrapper = styled.div`
+  display: flex;
+  width: 100%;
+  padding: 45px;
+  justify-content: center;
+  text-align: center;
+  align-items: center;
+`;
+
+const H2 = styled.h2`
+  font-size: 48px;
+`;
+
+const StyledLink = styled(Link)`
+  margin-right: 70px;
+  display: block;
+  height: 30px;
+`;
+
+const TeacherTitle = ({ title }) => (
```

Comment:
We use this component not only in teacher view, maybe we should change its name to `TableTitle`